sdk/textanalytics/ai-text-analytics/review/ai-text-analytics.api.md

```diff
@@ -247,6 +241,14 @@ export interface TextAnalyticsSuccessResult {
     readonly error?: undefined;
     readonly id: string;
     readonly statistics?: TextDocumentStatistics;
+    readonly warnings: TextAnalyticsWarning[];
+}
+
+// @public
+export interface TextAnalyticsWarning {
+    code: WarningCode;
+    message: string;
+    targetRef?: string;
```

Comment:
for 3.0 the service is not going to return any `targetRef`, so we decided not to include it in our SDK for P5 and eventually GA.